src/Dropdown/test/DropdownMenuSpec.js

```diff
@@ -234,6 +234,25 @@ describe('<Dropdown.Menu>', () => {
     assert.isNotNull(instance.querySelector('.rs-dropdown-item-submenu'));
   });
 
+  it('Should highlight menu item when hover', () => {
+    const { getByTestId } = render(
+      <DropdownMenu>
+        <DropdownItem data-testid="item-1">1</DropdownItem>
+        <DropdownItem data-testid="item-2">2</DropdownItem>
+        <DropdownItem data-testid="item-3">3</DropdownItem>
+        <DropdownItem data-testid="item-4">4</DropdownItem>
+      </DropdownMenu>
+    );
+
+    const menuItem = getByTestId('item-1');
+
+    act(() => {
+      Simulate.mouseOver(menuItem);
+    });
```

Comment:
Please use `@testing-library/user-event` as long as it's possible. You may use its `.hover()` method.